### DISCUSSION TITLE
Streamdeck sleepmode?

### DISCUSSION THREAD

**Discussion by Lavoslav** [bitfocus/companion#2815] [Q&A] [answered]
I can't find the option to put Streamdeck into sleep mode after X time of inactivity. Is there a way to make it so? 
I'm looking to just turn the baclight on the keys off.

**Comment by MarekPro1** [comment, marked as answer]
Hello I have a solution to that. I use trigger that is set to every hour set device to page -- and set brightnes to 0%. On page -- I have buttons that sets brightness to 100% and switch pages to the right one. Buttons are all over the page so when the stream deck is black I press any button and  it does right.

**Reply by Lavoslav** [reply]
Not native, but genius workaround. Ty!
